src/lib/routes/admin-api/client-metrics.ts

```diff
@@ -27,7 +31,11 @@ class ClientMetricsController extends Controller {
 
     async getRawToggleMetrics(req: Request, res: Response): Promise<void> {
         const { name } = req.params;
-        const data = await this.metrics.getClientMetricsForToggle(name);
+        const { hoursBack } = req.query;
+        const data = await this.metrics.getClientMetricsForToggle(
+            name,
+            ClientMetricsController.parseHoursBackQueryParam(hoursBack),
```

Comment:
What is the benefit of using a static method here instead of a private and `this`?

Reply:
True, no particular benefit. Just a preference for `static` when possible, seems simpler.

Reply:
I'm generally a fan of private static but this will be the first instance of a private static method in this codebase. If there's no reason not to, might be worth removing the static and using 'this' for consistencies sake